dvc/repo/experiments/__init__.py

```diff
@@ -628,7 +626,7 @@ def _reproduce(self, executors: dict, jobs: Optional[int] = 1) -> dict:
 
         Returns dict containing successfully executed experiments.
         """
-        result = {}
+        result: dict = {}
```

Comment:
I was not sure specifically what type it is. So, just left a `dict` there. Please make it more specific later.